src/class-scripts.php

```diff
@@ -124,6 +124,11 @@ public function enqueue_js_tracker(): void {
 			$js_api_key = "window.wpParselyApiKey = '" . esc_js( $this->parsely->get_api_key() ) . "';";
 			wp_add_inline_script( 'wp-parsely-loader', $js_api_key, 'before' );
 		}
+
+		if ( isset( $parsely_options['disable_autotrack'] ) && $parsely_options['disable_autotrack'] ) {
+			$should_load_autotrack = 'window.wpParselyDisableAutotrack = true';
+			wp_add_inline_script( 'wp-parsely-loader', $should_load_autotrack, 'before' );
```

Comment:
The `$should_load_autotrack` naming is a bit contrary to what the variable actually does. It would perhaps be better to name it `$should_disable_autotrack` or even `$disable_autotrack` for brevity.